native/cocos/bindings/event/EventDispatcher.cpp

```diff
@@ -289,6 +290,7 @@ void EventDispatcher::dispatchOrientationChangeEvent(int orientation) {
         args.emplace_back(se::Value(jsOrientationEventObj));
         func.toObject()->call(args, nullptr);
     }
+    EventDispatcher::dispatchCustomEvent("orientation_changed", 0);
```

Comment:
Please use macro for event name as other events.

Reply:
Code here is no more needed, I will delete